driver/asana_driver.py

```diff
@@ -158,18 +178,22 @@ def retrieve_metadata(self, milestone):
                     
                      # build document meta for attachment
                     doc = {
-                        'id' : attachment['id'], # TODO: verify id to be used
-                        'external_id' : attachment['id'], # TODO: verify id to be used
+                        'external_id' : attachment['id'],
```

Comment:
When creating attachments, add a key value to the doc `dirty: True`. This will signal the `retrieve_data` function to be called for that doc.